File: 18.py
k, n, w = map(int, input().split())
money = 0
if k>=1 and w<=1000:
  for i in range(1, w+1):
      money = money+(i*k)
  if money<=n:
      print(0)
  else:
    print(money-n)

File: 18.c
#include <stdio.h>

int main(){
  int k,n,w;
  scanf("%d %d %d",&k,&n,&w);
  int money=0;

  if(k>=1 && w<=1000){
    for (int i = 1; i <= w; i++)
    {
      money = money + (i*k);
    }
    if (money<=n){

      printf("%d",0);
    }
    else{

      printf("%d",(money-n));
    }
  }
  return 0;
}